src/components/layout.tsx

```diff
@@ -15,7 +15,7 @@ const sections: Section[] = [
   { title: "Contact", url: "https://www.ordina.nl/vakgebieden/scala/" },
 ];
 
-const Layout = ({ children }) => {
+const Layout: React.FC = ({ children }) => {
```

Comment:
I just use `const Layout: FC` instead of `React.FC`. I don't care either way, but shall we agree on a standard?